### DISCUSSION TITLE
Aktuell ist kein MQTT-Broker konfiguriert oder .... (2.0.5)

### DISCUSSION THREAD

**Discussion by bobmorane77** [camueller/SmartApplianceEnabler#314] [Installation und Betrieb]
Hi,
der Status meldet das es den MQTT nicht gibt.
Ich arbeite auf Dockerbasis mit eclipse-mosquitto.
Habe mir entsprechend ne yml zusammen gestrickt, das alles in einem Rutsch geht.
nutze die variante mit anonym erlaubt im mosquitto.

Log im mosquitto sagt ich kann den per browser erreichen - malformed logisch
und der SAE connected auch

1661507028: New connection from 192.168.0.112:65344 on port 1883.
1661507028: Client <unknown> disconnected due to malformed packet

1661506971: New client connected from 172.22.0.2:35766 as F-XXXXXXXX-XXXXXXXXXX-00-MeterReportingSwitch-3 (p2, c1, k60).
1661506971: New client connected from 172.22.0.2:35770 as F-XXXXXXXX-XXXXXXXXXX-00-S0ElectricityMeter-2 (p2, c1, k60).

Bug/Schönheitsfehler oder muss ich zwingend mit benutzername/pw arbeiten?

Danke

**Comment by camueller** [comment]
Es gibt eine 
[docker-compose.yml](https://github.com/camueller/SmartApplianceEnabler/blob/2.0/run/etc/docker/compose/docker-compose.yml), welche alle benötigten Dienste für SAE 2.0 startet.

In der [Dokumentation zur manuellen Installation](https://github.com/camueller/SmartApplianceEnabler/blob/2.0/doc/ManualInstallation_DE.md) ist auch die Konfiguration von Mosquitto beschrieben - standardmässig ohne Username/Passwort.

**Comment by bobmorane77** [comment]
mhm, ok danke die doku ist an mir vorbei gegangen.

generell aber gleiches Spiel. ich trage nur meine ips ein
stelle des in der appliance ein
steht auch im xml
        <MqttBroker host="192.168.0.223" port="1883"/>
        
Status sagt Broker fehlt, im mosquitto log sehe ich aber die appliance sich connecten

1661511158: New connection from 192.168.0.224:40324 on port 1883.
1661511158: New client connected from 192.168.0.224:40324 as F-76543218-170677000001-00-Appliance-0 (p2, c1, k60).

**Comment by camueller** [comment]
Ich kann leider nicht sagen, warum es bei Dir nicht fuktioniert. Bei mir läuft der SAE 2.0 seit Wochen mit o.g. Docker-Konfiguration ohne Probleme.
